packages/tasit-sdk/src/index.js

```diff
@@ -1,14 +1,22 @@
 "use strict";
 
-// TODO: Add Babel support
-
 import { Wallet } from "ethers/wallet";
 
-export const create = async () => {
+const create = async () => {
```

Comment:
@marcelomorgado Do you think it matters that we have this as an async function when Wallet.createRandom from ethers isn't?

Reply:
No, not really.